src/SocketIO.Serializer.SystemTextJson/SocketIO.Serializer.SystemTextJson.csproj

```diff
@@ -13,7 +13,7 @@
     <ProjectReference Include="..\SocketIO.Serializer.Core\SocketIO.Serializer.Core.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">
```

Comment:
I think we can use `<PackageReference Condition="'$(TargetFramework)' != 'net6.0' and '$(TargetFramework)' != 'net7.0' and '$(TargetFramework)' != 'net8.0'" Include="System.Text.Json" Version="8.0.3" />`

Reply:
For me I like having my conditions on temp groups but I can move it to the package.

I am not a fan of inverting the logic as shown due to if in the future we need to add net 9 the conditions need to be tweaked.

Reply:
~~I think we can use `<PackageReference Condition="'$(TargetFramework)' != 'net6.0' and '$(TargetFramework)' != 'net7.0' and '$(TargetFramework)' != 'net8.0'" Include="System.Text.Json" Version="8.0.3" />`~~

sorry, ignore it.

Current target framework is `netstandard2.0`, I don't know if the `<ItemGroup Condition="'$(TargetFramework)'=='netstandard2.0'">` and `<ItemGroup Condition="">` are equivalent

I will try pack it and test.

Reply:
Yes but up above I added net 6 as an additional target which doesn't need this dependency hence the conditionfor net standard 2.0